omod/src/main/java/org/openmrs/module/bedmanagement/rest/resource/AdmissionLocationResource.java

```diff
@@ -68,9 +67,19 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
         }
         if ((rep instanceof FullRepresentation)) {
             DelegatingResourceDescription description = new DelegatingResourceDescription();
+            description.addProperty("ward");
```

Comment:
Please change `"ward"` to `"location"`.

Reply:
It would not appropriate to change "ward" to "location" since at IPD section (angularJs) using ward keyword.
https://github.com/Bahmni/openmrs-module-bahmniapps/blob/e29c1939dbb1ef144a024be6b8c27bbead6ededd/ui/app/adt/controllers/wardController.js#L8
https://github.com/Bahmni/openmrs-module-bahmniapps/blob/e29c1939dbb1ef144a024be6b8c27bbead6ededd/ui/app/adt/controllers/wardController.js#L11
https://github.com/Bahmni/openmrs-module-bahmniapps/blob/e29c1939dbb1ef144a024be6b8c27bbead6ededd/ui/app/adt/controllers/wardController.js#L22
https://github.com/Bahmni/openmrs-module-bahmniapps/blob/e29c1939dbb1ef144a024be6b8c27bbead6ededd/ui/app/adt/controllers/wardLayoutController.js#L20
https://github.com/Bahmni/openmrs-module-bahmniapps/blob/e29c1939dbb1ef144a024be6b8c27bbead6ededd/ui/app/adt/controllers/wardListController.js#L16

Reply:
Very good point. Thanks @sanishmaharjan.

You see, this is exactly why consistent naming is important, from the first link that you provided for example:
```
$scope.ward.ward.name
```
This obviously doesn't mean much to an outsider.